### ISSUE TITLE
[MAS4.2.5] [Labels and Relationships -Add other service] Label is not defined descriptively for “URL”

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1812]
**Actual Result:**
Label is not defined descriptively for “URL” present in the Connect to a generic service dialog 

**Expected Result:**
Label should be defined descriptively for “URL” 
Example: Generic service URL

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2. Navigate to the bot explorer and select it 
3. Navigate to the services(+) and select it 
4.Menu gest opened 
5. Navigate to the Add other service and select it 
6. Dialog gets opened 
7.Observe the issue

**User Impact:**
Users who rely on AT will be affected if the label is not defined descriptively then the screen reader users will be affected 

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**Mas reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={07857fe1-d8ef-409e-a594-f440a8359566}
![MAS4 2 5_Add other service](https://user-images.githubusercontent.com/53518994/64062971-994a4a00-cc0b-11e9-90fc-2cb3088db33a.png)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue got fixed hence we are closing this bug.
Now Service Name and Service URL they are defined .
